### ISSUE TITLE
Doesn't work when using Shell History tools

### ISSUE THREAD

**Issue by ManveerBhullar** [pomdtr/tweety#14]
So when using this with either [Hishtory](https://github.com/ddworken/hishtory) or [RESH](https://github.com/curusarn/resh), both tools which replace your CNTRL-R history, i get a panic on startup.

Looks like Tweety doesnt have a $HOME.
```
panic: failed to retrieve config: failed to retrieve homedir: $HOME is not defined

goroutine 1 [running]:
github.com/ddworken/hishtory/client/hctx.MakeContext()
        github.com/ddworken/hishtory/client/hctx/hctx.go:123 +0x25c
github.com/ddworken/hishtory/client/cmd.glob..func8(0x140008c6700?, {0x1019c5b94?, 0x4?, 0x1019c5b44?})
        github.com/ddworken/hishtory/client/cmd/configGet.go:29 +0x1c
github.com/spf13/cobra.(*Command).execute(0x102ef9fa0, {0x102f52e40, 0x0, 0x0})
        github.com/spf13/cobra@v1.6.1/command.go:920 +0x654
github.com/spf13/cobra.(*Command).ExecuteC(0x102f00440)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.6.1/command.go:968
github.com/ddworken/hishtory/client/cmd.Execute()
        github.com/ddworken/hishtory/client/cmd/root.go:22 +0x24
main.main()
        github.com/ddworken/hishtory/hishtory.go:8 +0x1c
panic: failed to retrieve config: failed to retrieve homedir: $HOME is not defined

goroutine 1 [running]:
github.com/ddworken/hishtory/client/hctx.MakeContext()
        github.com/ddworken/hishtory/client/hctx/hctx.go:123 +0x25c
github.com/ddworken/hishtory/client/cmd.glob..func45(0x1400042c000?, {0x1018d1b94?, 0x4?, 0x1018d1b44?})
        github.com/ddworken/hishtory/client/cmd/saveHistoryEntry.go:55 +0x1c
github.com/spf13/cobra.(*Command).execute(0x102e0cce0, {0x140008df950, 0x3, 0x3})
        github.com/spf13/cobra@v1.6.1/command.go:920 +0x654
github.com/spf13/cobra.(*Command).ExecuteC(0x102e0c440)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.6.1/command.go:968
github.com/ddworken/hishtory/client/cmd.Execute()
        github.com/ddworken/hishtory/client/cmd/root.go:22 +0x24
main.main()
        github.com/ddworken/hishtory/hishtory.go:8 +0x1c
```

**Comment by pomdtr** [comment]
Interesting, it looks like bash does not automatically set HOME on startup. I'll investigate how other terminals handle it.

In the meantime, you can add the HOME env var to your tweety.json config

```json
{
   "defaultProfile": "bash",
   "profiles": {
      "bash": {
        "command": "/bin/bash",
        "args": [
          "--login"
        ],
        "env": {
          "HOME": "/Users/pomdtr"
        }
      }
   }
}
```

**Comment by pomdtr** [comment]
fixed in v0.8.3
